Velaptor/Observables/OpenGLContextReactor.cs

```diff
@@ -1,28 +1,33 @@
-﻿// <copyright file="OpenGLContextObservable.cs" company="KinsonDigital">
+﻿// <copyright file="OpenGLContextReactor.cs" company="KinsonDigital">
 // Copyright (c) KinsonDigital. All rights reserved.
 // </copyright>
 
 namespace Velaptor.Observables
 {
     // ReSharper disable RedundantNameQualifier
     using Velaptor.Observables.Core;
+    using Velaptor.Observables.ObservableData;
 
     // ReSharper restore RedundantNameQualifier
 
     /// <summary>
     /// Creates an observable to send push notifications of OpenGL events.
     /// </summary>
-    internal class OpenGLContextObservable : Observable<object>
+    internal class OpenGLContextReactor : Reactor<GLContextData>
     {
         /// <summary>
         /// Sends a push notification that the OpenGL context has been created.
         /// </summary>
         /// <param name="data">The data to send with the notification.</param>
```

Comment:
Add a summary section that the reactor(observable) auto unsubscribes itself after processing all of the push notifications.